scripts/shareCase.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+set -eu
+
+main() {
+    checkArgs "$@"
+
+    user=${1}
+    password=${2}
+    caseId=${3}
+    commaSeparatedUsersIds=${4}
+    idamUrl=${5:-"https://idam-api.platform.hmcts.net"}
+    fplUrl=${6:-"http://fpl-case-service-prod.service.core-compute-prod.internal"}
+
+    requestBody=$(buildPayload ${commaSeparatedUsersIds})
+    echo "Request payload: ${requestBody}"
+
+    accessToken=$(getAccessToken ${user} ${password})
+    response=$(shareCase ${caseId} ${accessToken})
+    echo "Response: ${response}"
+}
+
+checkArgs() {
+    if [ $# -lt 4 ]
+    then
+        echo "Usage: shareCase.sh <systemUserId> <systemUserPassword> <caseId> <comma separated list of userIds> [<idamUrl>] [<fplUrl>] "
+        exit 1
+    fi
+}
+
+buildPayload() {
+    usersIdsArray=(${1//,/ })
+    usersIds=$(printf ",\"%s\"" "${usersIdsArray[@]}")
+    usersIds=${usersIds:1}
+    requestBody="{\"ids\":[${usersIds}]}"
+    echo ${requestBody}
+}
+
+getAccessToken() {
```

Comment:
Can we not use  `fpla-docker/bin/utils/idam-authenticate.sh` folder to get the auth token?

Reply:
this is supposed to be standalone script, dont want devops to checkout any repo which contains dependencies

Reply:
Fair enough, thought that might be the case but wanted to raise it anyway